src/pymmcore_widgets/_shutter_widget.py

```diff
@@ -310,10 +311,8 @@ def _on_autoshutter_changed(self, state: bool) -> None:
         if self.autoshutter:
             with signals_blocked(self.autoshutter_checkbox):
                 self.autoshutter_checkbox.setChecked(state)
-        self.shutter_button.setEnabled(not state)
-
-        if state and self._mmc.isSequenceRunning():
-            self._mmc.stopSequenceAcquisition()
+        if self._mmc.getShutterDevice() == self.shutter_device:
+            self.shutter_button.setEnabled(not state)
```

Comment:
same here as above, should we just do this?
```python
self.shutter_button.setEnabled(not state)
```

Reply:
Same explanation as the other one, Autoshutter only controls one shutter at a time so only that shutter should be disabled.